examples/3DTilesOutliers.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>3D Tiles Outliers example</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/default.css" />
+  </head>
+  <body>
+    <script src="../packages/browser/dist/RUN_MODE/bundle.js"></script>
+
+    <script type="text/javascript">
+      udvizBrowser.FileUtil.loadMultipleJSON([
+        './assets/config/extent_lyon.json',
+        './assets/config/frame3D_planars.json',
+        './assets/config/layer/3DTiles.json',
+        './assets/config/layer/base_maps.json',
```

Comment:
maybe it's not necessary to load and add the base map since the example only show a 3DTiles feature ? 
If you think that base map should be loaded then I think elevation should be as well.

Reply:
since 3DTiles are loaded, without elevation they will float making it awkward. It ended up by adding the elevation layer also